ideas/172-topic-democracy.md

```diff
@@ -0,0 +1,83 @@
+
+When users don't vote and don't delegate, their influence would be delegated to SGT (Status Genesis Tokens) owners consensus.
+
+
+### Product Description
+
+- Users should be able to participate on decisions or delegate their influence. _Important to democracy._
+- Votes are counted as MiniMeToken balance of user at block voting end. _Important to not double tabulating without locking token transfers_ 
+- Voting or delegating must not cause direct risks to users balances. _Important to don't cause barriers into democracy process_
+- Users that delegates must be able to vote differently then his delegate. _Important to don't cause barriers into setting a delegate._
+- Delegates should be able to delegate influence delegated to them, as a delegation chain. _Important to inflience get into experts._
+- The voting process of proposals should be divided in two polls, first asking for approval votes, and second asking for reject votes.  _Important to prevent a big delegate changing decision at last moment._
+- Approval pool uses a "Executive Delegation" and rejection pool uses a "Veto Delegation". _Important to prevent bribe of delegates._
+- Specifics topics should have their own parent topic and specific delegation chain, which if unset by user should fallback to parent topic definition. _Important to influence get into specialists of the topic._
+- Users that didnt defined any delegate would by default delegate to SGT Topic Democracy. _Important to don't cause slowness into Status Network development._
+- SNT must be owned by SNT Topic Democracy. _Important to democracy._
+
+### Requirements & Dependencies
+
+- MiniMeToken
+
+### Minimum Viable Product
+<!-- Mandatory, completes the Idea in the fastest route possible, can be hacky, needed to feel progress. See https://imgur.com/a/HVlw3 -->
+Goal Date: <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: Topic Democracy as DAO Governance
+- Users can execution of proposals.
+- SNT is owned by topic democracy. 
+
+## Dates
+Goal Date: <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: Topic Democracy as "opinion" pool as experimental.
+- System is deployed in mainnet.
+- Users can cast opinion as vote through Status.
+- Users can delegate opinion through Status.
+- Inchain tabulation is not required.
+- Topic Democracy don't owns SNT.
+- Users are asked to approve proposal to upgrade into DAO Governance.
+- First DAO Governance can control network/assets by a multisig between current authority.
+
+## Success Metrics
+
+It's difficult to assume a success metric from the voting, as users that just didn't "touched" the product probably are delegating their trust to Status Genesis Token holders.
+
+However if we able to see if users look into the proposals and discuss about them in forums, would be a success signal.
```

Comment:
We could count number of participants maybe? And some measure of activity (even if delegated)

Reply:
Totally possible.